### PR TITLE
libsodium 1.0.19

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
`gstreamer` needs self-hosted Linux due to disk space.

**Comment by chenrui333** [comment]
proxgen test failure seems unrelated. we can start another run once merge the other big PRs:
- #151447 
- #151438

**Comment by EricFromCanada** [comment]
```
Error: 6 failed steps!
brew test --verbose fizz
brew test --verbose wangle
brew install --verbose --build-bottle edencommon
brew test --verbose fb303
brew linkage --test gstreamer
brew test --verbose proxygen
```
Each is expecting to find `libsodium.23.dylib`, but the new file path is `libsodium.26.dylib`.

**Comment by EricFromCanada** [comment]
```
% brew linkage --test gstreamer
Conflicting libraries:
  python@3.11
  python@3.12
```

**Review by SMillerDev — APPROVED** [review]

**Comment by chenrui333** [comment]
just `brew test --verbose gstreamer` test failure on linux build

**Comment by ZhongRuoyu** [comment]
`gstreamer` is a very popular formula and the failure should be easily fixed by adding a `python` runtime dependency. So let's fix it here and not remove its bottle.

```
  ==> /home/linuxbrew/.linuxbrew/Cellar/gstreamer/1.22.6_1/bin/gst-validate-launcher --usage
  /usr/bin/env: ‘python3’: No such file or directory
```

File: Formula/g/gstreamer.rb [hunk 1]
```diff
@@ -102,13 +104,13 @@ class Gstreamer < Formula
   end
 
   def python3
-    which("python3.11")
```
Comment (chenrui333) [inline Formula/g/gstreamer.rb:105]: why this change?

Comment (EricFromCanada) [reply Formula/g/gstreamer.rb:105]: https://github.com/Homebrew/homebrew-core/pull/142566#issuecomment-1784237103

Comment (EricFromCanada) [reply Formula/g/gstreamer.rb:105]: The current problem is `system python3, "-c"` failing because Python 3.12 is not the default (i.e. `python3` does not exist). Changing it to `system "python3.12", "-c"` should fix it, but the CI-long-timeout tag is currently occupied.

Comment (chenrui333) [reply Formula/g/gstreamer.rb:105]: noted, let me update the build for it

Comment (chenrui333) [reply Formula/g/gstreamer.rb:105]: actually no need to change to "python3.12", `which("python3.12")` should just work for gstreamer.

**Review by iMichka — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]
Just a couple of random git errors. Great job everyone! Merging.

**Comment by chenrui333** [comment]
- somehow missed the rev bump for php@8.1 https://github.com/Homebrew/homebrew-core/pull/153804
